### DISCUSSION TITLE
Typescript support

### DISCUSSION THREAD

**Discussion by ludeknovy** [canarytrace/documentation#140] [Synthetic]
Hi! Does canarytrace support executing (no need to transpile it and version it in a git repo) wdio tests written in TypeScript?

**Comment by rdpanek** [comment]
Hi @ludeknovy typescript is not supported in this time, but in future release probably with a new WDIO will be TS supported.

**Reply by ludeknovy** [reply]
Thanks! So I guess the new wdio v8 is not supported yet?

**Reply by rdpanek** [reply]
not yet, but in next release
